datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/controller/ExceptionHandlerApiController.java

```diff
@@ -0,0 +1,63 @@
+package de.adorsys.datasafe.rest.impl.controller;
+
+import de.adorsys.datasafe.rest.impl.exceptions.EmptyInputStreamException;
+import de.adorsys.datasafe.rest.impl.exceptions.FileNotFoundException;
+import de.adorsys.datasafe.rest.impl.exceptions.UserDoesNotExistsException;
+import de.adorsys.datasafe.rest.impl.exceptions.UserExistsException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@ControllerAdvice(basePackages = {"de.adorsys.datasafe.rest.impl"})
+@RestController
+@Slf4j
+public class ExceptionHandlerApiController {
+
+    @ExceptionHandler({UserDoesNotExistsException.class})
+    public ResponseEntity<String> handleUserDoesNotExistsException(UserDoesNotExistsException ex) {
+        log.debug("User does not exists exception: " + ex.getMessage());
```

Comment:
Slf4j has {} string interpolation, also you can log exception directly by
log.debug("User already exists exception", ex);

Reply:
ok